.vscode/extensions.json

```diff
@@ -0,0 +1,12 @@
+{
+  "recommendations": [
+    "charliermarsh.ruff",
+    "magicstack.magicpython",
+    "ms-python.debugpy",
+    "ms-python.mypy-type-checker",
```

Comment:
I have been using `matangover.mypy` and was quite happy with it, will try the MS version tomorrow morning and see if it is better or worse.